aio/content/examples/getting-started/src/app/shipping/shipping.component.ts

```diff
@@ -1,8 +1,9 @@
 // #docplaster
 // #docregion imports
-import { Component } from '@angular/core';
+import { Component , OnInit  } from '@angular/core';
```

Comment:
Nit: Inconsistent spacing:

```suggestion
import { Component, OnInit  } from '@angular/core';
```